pkg/hub/localizer/patch_test.go

```diff
@@ -52,19 +65,8 @@ hole: black
 		want      []byte
 	}{
 		{
-			name: "Helm",
-			original: []byte(`{
-				"kind": "Guess",
-				"address": {
-					"city": "Nantucket",
-					"street": "123 Spouter Inn Ct."
-				},
-				"boat": "pequod",
-				"details": {
-					"friends": ["Tashtego"]
-				},
-				"name": "Ishmael"
-			}`),
```

Comment:
No need to define a new test. Just insert an empty `OverrideConfig` into exisitng overrides.